lib/krikri/enrichments/timespan_split.rb

```diff
@@ -61,7 +61,10 @@ def parse_labels(labels)
 
     def span_from_date(date)
       return [nil, nil] if date.nil?
-      return [date, date] if date.is_a? Date
+      if date.is_a?(Date)
+        return [date, date] if date.precision == :day
+        return [date, (date.succ - 1)]
```

Comment:
I looked up the `Date` documentation to find the definition of `#succ`, which is the succeeding date.  Isn't the succeeding date minus one day the same as `date`?  I could be missing something, but ... my main point: a comment might help to make this clear :-)